Fix leftover 'Compiler' occurrences by renaming to 'Sass'

Ragmaanir opened the pull request:
Not 100% sure that this is correct, but it works for me.
Without this change i get:

```
In lib/sass/src/compiler.cr:144:5

 144 | Compiler.compile_file(file, **options)
       ^-------
Error: undefined constant Compiler
```

Ragmaanir committed:
Fix leftover 'Compiler' occurrences by renaming to 'Sass' and add regression tests

straight-shoota commented:
This looks about right.

It seems specs are only testing the class methods. To make sure it keeps working, could you add two specs to `compiler_spec` for this? They should probably just do `Sass.new.compile_file` instead of `Sass.compile_file`.

Ragmaanir commented:
Ok, i added two tests now (just copied the simplest tests for the class methods). I also changed the names of the describe blocks for the class-methods from `"#compile_file"` to `".compile_file"`.

straight-shoota commented:
Thank you
